packages/js/product-editor/src/blocks/generic/linked-product-list/editor.scss

```diff
@@ -0,0 +1,143 @@
+.wp-block-woocommerce-product-linked-list-field {
+	display: flex;
+	flex-direction: column;
+	gap: $grid-unit-30;
+
+	[role="table"] {
+		width: 100%;
+
+		[role="rowheader"],
+		[role="row"] {
+			display: grid;
```

Comment:
Did you consider having the whole table be `grid`?

Reply:
Yes and it does not work with a11y.

Reply:
Interesting. I'd love to learn more about why this didn't work, to increase my understanding!